src/pygama/raw/build_raw.py

```diff
@@ -17,8 +19,9 @@
 log = logging.getLogger(__name__)
 
 
-def build_raw(in_stream, in_stream_type=None, out_spec=None, buffer_size=8192,
-              n_max=np.inf, overwrite=True, **kwargs):
+def build_raw(in_stream: int, in_stream_type: str = None,
+              out_spec: str | dict | RawBufferLibrary = None, buffer_size: int = 8192,
+              n_max: int = np.inf, overwrite: bool = True, **kwargs) -> None:
```

Comment:
np.inf is a float -- is it okay to say n_max must be an int and give it a float default value? Or is there an int max that we can change it to?